ruby-challenges.rb

```diff
@@ -3,32 +3,81 @@
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
-reposts1 = 7
-# Expected output: '7 is odd'
-reposts2 = 42
-# Expected output: '42 is even'
-reposts3 = 221
-# Expected output: '221 is odd'
+# Create a method called "number_checker"
+# create a conditional statment that will determine if the number is odd or even using the modulo method
+# Return a particular statment using string interpoloation based on if the number is even or odd
 
+# def number_checker (num)
+#     if num % 2 == 0 
+#         '#{num} is even'
+#     else 
+#         '#{num} is odd'
+#     end
+
+# end
+
+
```

Comment:
Nice use of the modulo! Are there any built-in methods you can use to make this even simpler?